app/sidecar/rev-a.toml

```diff
@@ -740,6 +740,22 @@ device = "pca9538"
 description = "Front IO GPIO expander"
 removable = true
 
+[[config.i2c.devices]]
+bus = "front_io"
+address = 0b0001_010
+device = "pca9956b_left"
```

Comment:
`device` should be the device name (without qualifiers), i.e. `pca9956b`.  Add a `name` field to distinguish them (here and below), e.g.
```toml
name = "front_leds_left"
```

After this, you'll need to edit `drv-transceivers-server` to use the new names, i.e.
```diff
diff --git a/drv/transceivers-server/src/main.rs b/drv/transceivers-server/src/main.rs
index a9b1a5fa..52f7f67d 100644
--- a/drv/transceivers-server/src/main.rs
+++ b/drv/transceivers-server/src/main.rs
@@ -235,8 +235,8 @@ fn main() -> ! {

         let transceivers = Transceivers::new(FRONT_IO.get_task_id());
         let leds = Leds::new(
-            &i2c_config::devices::pca9956b_left_front_io(I2C.get_task_id())[0],
-            &i2c_config::devices::pca9956b_right_front_io(I2C.get_task_id())[0],
+            &i2c_config::devices::pca9956b_front_leds_left(I2C.get_task_id()),
+            &i2c_config::devices::pca9956b_front_leds_right(I2C.get_task_id()),
         );

         let mut server = ServerImpl {
```